aksolve/client/aksolve/symmlq.py

```diff
@@ -0,0 +1,242 @@
+def symmlq(
```

Comment:
This function looks to have less natural breakpoints than the other, but I definitely think we could sub-function some elements out for readability here.